frontend/src/pages/inventory/csv-impex/csv-export.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Button } from '@material-ui/core';
+import { saveAs } from 'file-saver';
+import { jsonToCSV } from 'react-papaparse';
+
+export default function ExportButton() {
+  const toCSV = async () => {
+    const request = await fetch('http://localhost:5000/good/', {
```

Comment:
Could you use the api from the config a bit like how tommy did it here:
* https://github.com/Talos10/SOEN390-team-9/blob/ec5a9eaeffb672462991ec6dd24bb1db997f790f/frontend/src/utils/api.tsx#L5
* https://github.com/Talos10/SOEN390-team-9/blob/ec5a9eaeffb672462991ec6dd24bb1db997f790f/frontend/src/pages/login/resetPassword/ResetPassword.tsx#L7

This will make it easier when we will host the frontend using a hosting service

Reply:
All changes were made ^^